frontend/e2e-test/test/e2e/specs/1_application/application.spec.ts

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import Home from '../../pages/home'
+import {
+  initializeAreaAndPersonData,
+  AreaAndPersonFixtures
+} from '../../dev-api/data-init'
+import {
+  applicationFixture,
+  applicationFixtureId,
+  uuidv4
+} from '../../dev-api/fixtures'
+import { logConsoleMessages } from '../../utils/fixture'
+import { deleteApplication, insertApplications } from '../../dev-api'
+import { enduserRole, seppoAdminRole } from '../../config/users'
+import EnduserPage from '../../pages/enduser/enduser-navigation'
+import ApplicationReadView from '../../pages/employee/applications/application-read-view'
+
+const home = new Home()
+const enduserPage = new EnduserPage()
+const applicationReadView = new ApplicationReadView()
+
+let fixtures: AreaAndPersonFixtures
+let cleanUp: () => Promise<void>
+
+fixture('Employee reads applications')
```

Comment:
I think you should rename this fixture and the whole file probably as well.

Reply:
My intention was to actually have this as the new home for all application tests as it uses the latest and greatest tricks to keep the tests fast and simple (like using application fixtures instead of creating the application via UI), but yes, that was BDUF thinking. Renamed.

Reply:
Ah, I see.